src/colmap/scene/reconstruction.h

```diff
@@ -59,6 +59,8 @@ class DatabaseCache;
 class Reconstruction {
  public:
   Reconstruction();
+  // Update the camera pointers for each image at copy constructor.
+  inline Reconstruction(const Reconstruction& recon);
```

Comment:
I suggest to not inline but move the implementation to the .cc file.

Reply:
Done.